build.gradle.kts

```diff
@@ -193,7 +193,7 @@ subprojects {
     extra["logbackVersion"] = "1.2.3"
     extra["mavenVersion"] = "3.5.0"
     extra["ownerVersion"] = "1.0.12"
-    extra["pmeVersion"] = "4.5"
+    extra["pmeVersion"] = "4.6-SNAPSHOT"
```

Comment:
Needs final version?

Reply:
Yes it will eventually. I'm ok with snapshot during development.